src/alf/fonts.ts

```diff
@@ -78,8 +104,19 @@ export function applyFonts(style: TextStyle, fontFamily: 'system' | 'theme') {
  * This is used for both web fonts and native fonts.
  */
 export function DO_NOT_USE() {
-  return useFonts({
-    InterVariable: require('../../assets/fonts/inter/InterVariable.ttf'),
-    'InterVariable-Italic': require('../../assets/fonts/inter/InterVariable-Italic.ttf'),
-  })
+  return useFonts(
+    isAndroid
+      ? {
+          'Inter-Regular': require('../../assets/fonts/inter/Inter-Regular.otf'),
+          'Inter-Italic': require('../../assets/fonts/inter/Inter-Italic.otf'),
+          'Inter-Bold': require('../../assets/fonts/inter/Inter-SemiBold.otf'),
+          'Inter-BoldItalic': require('../../assets/fonts/inter/Inter-SemiBoldItalic.otf'),
+          'Inter-Black': require('../../assets/fonts/inter/Inter-ExtraBold.otf'),
+          'Inter-BlackItalic': require('../../assets/fonts/inter/Inter-ExtraBoldItalic.otf'),
+        }
+      : {
+          InterVariable: require('../../assets/fonts/inter/InterVariable.ttf'),
+          'InterVariable-Italic': require('../../assets/fonts/inter/InterVariable-Italic.ttf'),
+        },
+  )
 }
```

Comment:
Can we platform split this? Pretty sure this will bloat our bundle sizes on web.

Reply:
Fixed. Yeah the bot here was reporting totally assets, but those fonts wouldn't have actually loaded. Much better now though, thanks for the bump 👍